pkg/log/log.go

```diff
@@ -0,0 +1,12 @@
+package log
+
+import (
+	"log"
+	"os"
+)
+
+// ErrLogger is a logger than write to /dev/stderr
+var ErrLogger = log.New(os.Stderr, "", log.LstdFlags)
+
+// OutLogger is a logger than write to /dev/stdout
+var OutLogger = log.New(os.Stdout, "", log.LstdFlags)
```

Comment:
Any way we can use standard logger levels interface for these (`info` and `error`) and decide in this module here if we want to dump stuff to stderr or stdout?

Reply:
Thought about doing this earlier but realised there is no standard logger interface and so I'd have to create free function wrappers for all those methods. It seemed more cumbersome that way. This current implementation makes for an arguably clunky UX but an explicit one. I think I can live with that